sentry-core/src/main/java/io/sentry/core/Hub.java

```diff
@@ -77,14 +77,18 @@ public SentryId captureException(Throwable throwable) {
   public void close() {
     // Close the top-most client
     StackItem item = stack.peek();
-    item.client.close();
+    if (item != null) {
+      item.client.close();
+    }
     isEnabled = false;
   }
 
   @Override
   public void addBreadcrumb(Breadcrumb breadcrumb) {
     StackItem item = stack.peek();
-    item.scope.addBreadcrumb(breadcrumb);
+    if (item != null) {
+      item.scope.addBreadcrumb(breadcrumb);
```

Comment:
Same here. Item should never be null, neither scope (which isn't checked right now)

Reply:
`scope` can't be null, it's not a parameter but created by us on the constructor